interactive_engine/compiler/src/main/java/com/alibaba/graphscope/common/ir/meta/fetcher/DynamicIrMetaFetcher.java

```diff
@@ -81,7 +81,7 @@ private synchronized void syncMeta() {
             IrMeta meta = this.reader.readMeta();
             logger.debug(
                     "schema from remote: {}",
-                    (meta == null) ? null : meta.getSchema().getSchemaSpec(Type.IR_CORE_IN_JSON));
+                    (meta == null) ? null : meta.getSchema().getSchemaSpec(Type.FLEX_IN_JSON));
```

Comment:
We need to use `FLEX_IN_JSON`, `IR_CORE_IN_JSON` doesn't support current data type system.

Reply:
It seems that use FLEX_IN_JSON will cause other CI failure. This debug log is removed.